.gitignore

```diff
@@ -87,4 +87,11 @@ siglens.db
 
 
 # Alerting Scale testing output
-vector_log_lib/
\ No newline at end of file
+vector_log_lib/
+
+
+# MacOS
+.DS_Store
+
+# VS Code
+.vscode/
```

Comment:
We already have these (lines 20 and 50). Were the existing rules not working for you?

Reply:
I could see that the .vscode folder was being added to the commit. I see that the .gitignore has .vscode/* which ignores files inside it.

Will remove it.